sentry-ruby/Gemfile

```diff
@@ -3,7 +3,13 @@ git_source(:github) { |name| "https://github.com/#{name}.git" }
 
 gem "sentry-ruby", path: "./"
 
-gem "rack" unless ENV["WITHOUT_RACK"] == "1"
+unless ENV["WITHOUT_RACK"] == "1"
```

Comment:
Do you think we can use `RACK_VERSION=0` for this?

Reply:
yes we can simplify!